saleor/graphql/core/types/common.py

```diff
@@ -140,6 +140,14 @@ class OrderError(Error):
     order_line = graphene.ID(
         description="Order line ID which causes the error.", required=False,
     )
+    variants = graphene.List(
+        graphene.NonNull(graphene.ID),
+        description=(
+            "List of product variants product variant that are not published "
```

Comment:
This field can be used in all order mutations so I would say "List of product variants that are associated with the error".